Engine/Engine.cs

```diff
@@ -541,4 +545,23 @@ private static MarketHoursDatabase StaticInitializations()
         }
 
     } // End Algorithm Node Core Thread
+
+    public class InitializeException : Exception
+    {
+        private readonly string stackTrace;
```

Comment:
`stackTrace` not following code style, should be `_stackTrace`. Can make this class private 👌 . Should add some simple xml documentation about this class